hatchery/nextflow.go

```diff
@@ -354,8 +354,7 @@ func createNextflowUserResources(userName string, bucketName string, batchComput
                 			}
             			}
 
-			},
-			{
+			}
```

Comment:
want to note that this commit allows me to spin up a workspace successfully with this branch (previous commits failed for some reason), but this does not restrict the users to only run a specific container. Subsequent tests will be to add batch user nobody back in and move the image equals condition up, see https://docs.aws.amazon.com/batch/latest/userguide/ExamplePolicies_BATCH.html#iam-example-job-def